config/application.yml.default

```diff
@@ -310,6 +310,7 @@ s3_reports_enabled: false
 saml_secret_rotation_enabled: false
 seed_agreements_data: true
 service_provider_request_ttl_hours: 24
+sender_id_countries: 'BY,EG,FR,JO,PH,TH'
```

Comment:
Yeah, probably a good idea

Reply:
The reason it was originally a constant was so that we could use it for understanding if we had SMS support in a country, which is run locally, not in a deploy env. We might want to undo/remove that bit entirely from there then?

Reply:
Maybe we update the parsing to have `supports_sms` / `sms_sender_id_required` as two separate properties so that we can do the capability detection at runtime in prod?

Reply:
Hm, I see now. Is it worth trying to make this configurable?

Reply:
I am not sure! I think maybe the way we use it in `config/initializers/telephony.rb` could be migrated to a config? but this script doesn't benefit the same way